common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -19,13 +19,14 @@ class ProblemsTest(UniqueCourseTest):
     """
     Base class for tests of problems in the LMS.
     """
-    USERNAME = "joe_student"
-    EMAIL = "joe@example.com"
-    PASSWORD = "keep it secret; keep it safe."
 
     def setUp(self):
         super(ProblemsTest, self).setUp()
 
+        self.username = "test_student_{uuid}".format(uuid=self.unique_id[0:8])
```

Comment:
@benpatterson I've updated this so that the user checked for in the events validation part of the tests is unique.